packages/@lwc/engine/src/framework/modules/props.ts

```diff
@@ -60,9 +54,6 @@ function update(oldVnode: VNode, vnode: VNode) {
             isFirstPatch ||
             cur !== (isLiveBindingProp(sel as string, key) ? elm[key] : (oldProps as any)[key])
         ) {
-            if (isCustomElement) {
-                prepareForPropUpdate(vm); // this is just in case the vnode is actually a custom element
```

Comment:
not sure how is that we can remove this? did we removed the error on the other end for non-prepared mutations?

Reply:
I think this is some legacy code that has not been updated during https://github.com/salesforce/lwc/pull/1134. If you look into the [`prepareForPropsUpdate`](https://github.com/salesforce/lwc/pull/1650/files#diff-57b9dec7c1ef7c4d63147ac2a47e1772L28-L32), you can see that it is an empty function.